src/Ledger/Certs.lagda

```diff
@@ -466,15 +466,12 @@ data _⊢_⇀⦇_,CERTBASE⦈_ where
         validVoteDelegs  = voteDelegs ∣^ (  mapˢ (credVoter DRep) (dom dReps)
                                         ∪ fromList (noConfidenceRep ∷ abstainRep ∷ []) )
     in
-    ∙ filter isKeyHash wdrlCreds ⊆ dom voteDelegs
-    ∙ mapˢ (map₁ stake) (wdrls ˢ) ⊆ rewards ˢ
```

Comment:
I see you moved the check `filter isKeyHash wdrlCreds ⊆ dom voteDelegs` into a premise of the `LEDGER` rule, but where did you put the `mapˢ (map₁ stake) (wdrls ˢ) ⊆ rewards ˢ` check.  I couldn't find it.  If you think there's a reason to not have it, please add a comment to the change.